vaadin-grid-flow/src/main/resources/META-INF/resources/frontend/gridConnector.js

```diff
@@ -5,7 +5,7 @@ window.Vaadin.Flow.gridConnector = {
       return;
     }
 
-    Vaadin.Grid.ItemCache.prototype.ensureSubCacheForScaledIndex = function(scaledIndex) {
+    grid._cache.ensureSubCacheForScaledIndex = function(scaledIndex) {
```

Comment:
I think this will only patch the first level cache. But with hiearachical grid, all the levels have their own cache instances and those remain unpatched.